frontend/src/context/userStateUtils.ts

```diff
@@ -9,19 +9,18 @@ export const getExtendedOrg = (
 ) => {
   let current: CurrentOrganization | null =
     org ?? user?.roles[0]?.organization ?? null;
-  if (!current) {
-    return null;
+  if (current) {
+    current.userIsAdmin =
+      user?.userType === 'globalAdmin' ||
+      user?.roles.find((role) => role.organization.id === current?.id)?.role ===
```

Comment:
What's the difference between `current` and `org` here? Is `org` somehow cached?